public/Get-DbaDbUserDefinedTableType.ps1

```diff
@@ -74,7 +83,11 @@ function Get-DbaDbUserDefinedTableType {
                 continue
             }
 
-            foreach ($tabletype in $db.UserDefinedTableTypes) {
+            if ($Type) {
+                $userDefinedTableTypes = $db.UserDefinedTableTypes | Where-Object Name -in $Type
+            } else { $userDefinedTableTypes = $db.UserDefinedTableTypes }
```

Comment:
thanks so much for the PR and functionality! This should be on the next line, I'll go ahead and add it 👍🏼

Reply:
Thanks for reviewing. I missed the bit to format it using dbatools function.